pvanalytics/system.py

```diff
@@ -1 +1,121 @@
 """Functions for identifying system characteristics."""
+import pandas as pd
+import numpy as np
+import pvlib
+from pvanalytics.util import _fit, _group
+
+
+def _peak_times(data):
+    minute_of_day = pd.Series(
+        data.index.hour * 60 + data.index.minute,
+        index=data.index
+    )
+    peak_minutes = _group.by_day(data).apply(
+        lambda day: pd.Timedelta(
+            minutes=round(
+                _fit.quadratic_vertex(
+                    x=minute_of_day[day.index],
+                    y=day,
+                )
+            )
+        )
+    )
+    return pd.DatetimeIndex(
+        np.unique(data.index.date),
+        tz=data.index.tz
+    ) + peak_minutes
+
+
+def infer_orientation_solarnoon(power_or_poa, sunny, tilts,
```

Comment:
how about `_daily_peak`?  If you agree, the `Solar noon is estimated` -> `The time of the daily peak is estimated`